pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <assertj.version>2.6.0</assertj.version>
-        <item-renderer.version>3.0.6</item-renderer.version>
+        <item-renderer.version>3.0.7</item-renderer.version>
```

Comment:
Is this right?  Won't the calculator code need to be released before you can bump?